sdk/endpoints/online/amlarc/online-endpoints-safe-rollout.ipynb

```diff
@@ -0,0 +1,520 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#enter details of your AML workspace\n",
+    "subscription_id = '<SUBSCRIPTION_ID>'\n",
+    "resource_group = '<RESOURCE_GROUP>'\n",
+    "workspace = '<AML_WORKSPACE_NAME>'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#get a handle to the workspace\n",
+    "ml_client = MLClient(InteractiveBrowserCredential(), subscription_id, resource_group, workspace)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Configure Kubernetes cluster for machine learning\n",
+    "Next, configure Azure Kubernetes Service (AKS) and Azure Arc-enabled Kubernetes clusters for inferencing machine learning workloads.\n",
+    "There're some prerequisites for below steps, you can check them [here](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-attach-arc-kubernetes).\n",
+    "\n",
+    "## 2.1 Connect an existing Kubernetes cluster to Azure Arc\n",
+    "This step is optional for [AKS cluster](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough).\n",
+    "Follow this [guidance](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster) to connect Kubernetes clusters.\n",
+    "\n",
+    "## 2.2 Deploy Azure Machine Learning extension\n",
+    "Depending on your network setup, Kubernetes distribution variant, and where your Kubernetes cluster is hosted (on-premises or the cloud), choose one of options to deploy the Azure Machine Learning extension and enable inferencing workloads on your Kubernetes cluster.\n",
+    "Follow this [guidance](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-attach-arc-kubernetes?tabs=studio#inferencing).\n",
+    "\n",
+    "## 2.3 Attach Arc Cluster\n",
+    "You can use Studio, Python SDK and CLI to attach Arc cluster to Machine Learning workspace.\n",
+    "Below code shows the attachment of AKS that the compute type is managedClusters. For Arc connected cluster, it should be connectedClusters.\n",
+    "Follow this [guidance](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-attach-arc-kubernetes?tabs=studio#attach-arc-cluster) for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ml.entities import Compute\n",
+    "\n",
+    "compute_params = [{'name': '<COMPUTE_NAME>'}, {'type': 'kubernetes'}, {'resource_id': '/subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP>/providers/Microsoft.ContainerService/managedClusters/<CLUSTER_NAME>'}]\n",
```

Comment:
I would advise we give two examples of resource id for raw aks cluster (managed cluster) and connected cluster. In your example, it's for raw aks cluster.

Reply:
added